### DISCUSSION TITLE
Don't send x-rate-limit information in response headers

### DISCUSSION THREAD

**Discussion by siobhan1byrne** [vladimir-bukhtoyarov/bucket4j#175] [Q&A] [answered]
Using SpringBoot Bucket4J through application.properties only. 
How do I stop the x-rate limit headers from being added to responses
x-rate-limit-remaining: 4 # in Response 200
 x-rate-limit-retry-after-seconds # in Response 429

Don't want either of those returned for security reasons. 
Here are my properties.

bucket4j.filters[1].cache-name=buckets
bucket4j.filters[1].filter-method=servlet
bucket4j.filters[1].http-response-body={ "message": "Too many requests" }
bucket4j.filters[1].url=/v1/*.*
bucket4j.filters[1].strategy=first
bucket4j.filters[1].rate-limits[0].bandwidths[0].capacity=5
bucket4j.filters[1].rate-limits[0].bandwidths[0].time=1
bucket4j.filters[1].rate-limits[0].bandwidths[0].unit=minutes
bucket4j.filters[1].rate-limits[0].bandwidths[0].fixed-refill-interval=1
bucket4j.filters[1].rate-limits[0].bandwidths[0].fixed-refill-interval-unit=minutes

**Comment by vladimir-bukhtoyarov** [comment, marked as answer]
Hello @siobhan1byrne 

Please route your question directly to https://github.com/MarcGiffing/bucket4j-spring-boot-starter
